content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/print_on_validation.rst

```diff
@@ -0,0 +1,156 @@
+=================================
+Print delivery slip on validation
+=================================
+
+Automatically print delivery slips that contain details about the package's recipient, contents, or
+handling instructions.
+
+In Odoo, the PDFs of the following documents can be configured to print upon validating a certain
```

Comment:
What "certain" kind of operation? Could clarify here

Reply:
Elaborated to include examples of operation types!